src/LiveComponent/doc/index.rst

```diff
@@ -1,11 +1,6 @@
 Live Components
 ===============
 
-.. caution::
```

Comment:
Same here :)

Reply:
We don't add it to symfony/symfony, so I'm inclined to follow that.